model/span.go

```diff
@@ -30,6 +30,8 @@ const (
 	//FirehoseFlag is the bit in Flags in order to define a span as a firehose span
 	FirehoseFlag = Flags(8)
 
+	SpanKindUnknown = ""
```

Comment:
Does this need to be exported? 

Also - could you document why the empty string was chosen instead of `unknown`?

Reply:
We decide to use empty string so that the endpoint can be backward compatible: GET /api/v1/operations?serivce={}
If spanKind is not present, the operation index will use empty string to query Cassandra to return operations names of all span kind.

I don't feel it's right to document the reason here in the model as it's related to the API.

Do you think "// Used to indicate the span.kind tag is missing" is enough?

Reply:
as far as this API goes, you don't need to have the constant, you can simply return a boolean flag
```
GetSpanKind() (spanKind string, found bool)
```